src/reducers/tags.js

```diff
@@ -0,0 +1,64 @@
+const initialState = [
+  {
+    id: 0,
+    categoryId: 1,
+    value: 'Javascript',
+    companies: [
+      {companyId: 0},
+      {companyId: 1},
+      {companyId: 2}
+    ]
+  },
+  {
+    id: 1,
+    categoryId: 1,
+    value: 'PHP',
+    companies: [
+      {companyId: 3},
+      {companyId: 4},
+      {companyId: 5}
+    ]
+  },
+  {
+    id: 2,
+    categoryId: 2,
+    value: 'Nairobi',
+    companies: [
+      {companyId: 0}
+    ]
+  },
+  {
+    id: 3,
+    categoryId: 2,
+    value: 'Auckland',
+    companies: [
+      {companyId: 4}
+    ]
+  },
+  {
+    id: 4,
+    categoryId: 3,
+    value: 'ReactJS',
+    companies: [
+      {companyId: 6}
+    ]
+  },
+  {
+    id: 5,
+    categoryId: 3,
+    value: 'AngluarJS',
+    companies: [
+      {companyId: 0}
+    ]
+  }
+]
```

Comment:
Looks good! Please change the initial state `companies` to match these tags.
I.e. Company id=0 will start with a `tags` array equal to `[0]`